src/main/java/org/truffleruby/interop/OutgoingForeignCallNode.java

```diff
@@ -113,6 +123,14 @@ protected OutgoingNode createHelperNode(int argsLength) {
             return new CallOutgoingNode(argsLength);
         } else if (name.equals("new")) {
             return new NewOutgoingNode(argsLength);
+        } else if (name.equals("to_a") || name.equals("to_ary")) {
+            return new ToAOutgoingNode();
+        } else if (name.equals("respond_to?")) {
+            return new RespondToOutgoingNode();
+        } else if (name.equals("inspect")) {
+            return new InspectOutgoingNode();
+        } else if (name.equals("send")) {
```

Comment:
Lots of other methods make sense - for example `method` that gives you a method that will send the interop message when you do call it.

But there's a long list of these so I didn't want to add each one until it is needed for something.